### PR TITLE
RELATED: ONE-4414 Extend structure of widget by properties+references

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R209](https://confluence.intgdc.com/display/Dev/R209).
This release is **zero downtime** release.
Code drop: 2020-07-16 00:00.

**Comment by yenkins** [comment]
Story [ONE-4414](http://jira.intgdc.com/browse/ONE-4414) Manual column resizing in KD
	Status: Hacking
	Priority: Major
	Assignee: Ivan Nejezchleb

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/123/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/208/ : SUCCESS in 50s

File: src/VisualizationObject.ts [hunk 1]
```diff
@@ -184,7 +184,14 @@ export namespace VisualizationObject {
         periodsAgo: number;
     }
 
-    export interface IVisualizationObject {
+    export interface IObjectWithProperties {
+        content: {
+            properties?: string;
+            references?: VisualizationObject.IReferenceItems;
+            [keys: string]: any;
```
Comment (pbenes) [inline src/VisualizationObject.ts:191]: `keys: string]: any;` is strange here. Do we really need it?

Comment (ivanmjartan) [reply src/VisualizationObject.ts:191]: Yes there are serval other items AD very specific

Comment (ivanmjartan) [reply src/VisualizationObject.ts]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/127/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/213/ : SUCCESS in 31s

File: src/VisualizationObject.ts (hunk 1 above)
Comment (ivanmjartan) [reply src/VisualizationObject.ts:191]: reverted back

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/128/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/214/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/129/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/215/ : SUCCESS in 1m 06s

**Comment by yenkins** [comment]
Currently destined for [release R210](https://confluence.intgdc.com/display/Dev/R210).
This release is **zero downtime** release.
Code drop: 2020-07-30 00:00.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/133/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/219/ : SUCCESS in 34s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/43/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/220/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-typings-fossa-scanning-zuul-el7/7/ : SUCCESS in 13s (non-voting)

**Pull request merged**